modules/porous_flow/doc/content/source/auxkernels/PorousFlowPropertyAux.md

```diff
@@ -22,6 +22,7 @@ following properties are available using the `property` input parameter:
 - `permeability`
 - `hystersis_order`
 - `hysteresis_saturation_turning_point`
+- [`hysteretic_info`](PorousFlowHystereticInfo.md)
```

Comment:
This doesn't look like a link on the website, so no-one will click it.

Reply:
made link clear